lib/class-wp-rest-url-details-controller.php

```diff
@@ -218,7 +218,7 @@ private function get_remote_url( $url ) {
 	 * @return string the title tag contents (maybe empty).
 	 */
 	private function get_title( $html ) {
-		preg_match( '|<\s*title[^>]*>(.*?)<\s*/\s*title>|is', $html, $match_title );
+		preg_match( '|<title[^>]*>(.*?)<\s*/\s*title>|is', $html, $match_title );
```

Comment:
@getdave I removed the whitespace between the opening `<` and the keyword `title`. Why? Whitespace in this spot causes the element to be malformed. Checking in multiple browsers, it was not autofixed by the browser.

What about the closing tag? The browser autofix any whitespace. Thank you browser.